ambari-server/src/main/java/org/apache/ambari/server/api/services/AlertTargetService.java

```diff
@@ -54,7 +54,7 @@
  * targets.
  */
 @Path("/alert_targets/")
-@Api(value = "/alerts", description = "Endpoint for alert specific operations")
+@Api(value = "/Alerts", description = "Endpoint for alert specific operations")
 public class AlertTargetService extends BaseService {
```

Comment:
I think the leading slash is also not needed, so it should be just "Alerts" (other @Api declarations don't use slashes).